cypress/integration/chi-js/mobile-nav.spec.js

```diff
@@ -0,0 +1,175 @@
+      cy.get('.chi-mobile-nav__user')
+        .should('be.visible')
+    });
+    
+  });
+  describe('open and close of button-site-menu2', () => {
+    it('click on button-site-menu2 open the dropdown items', () => {
+      cy.get('#button-site-menu2')
+        .click()
+        .wait(1000)
+      cy.get('#dropdown-2-menu')
+        .should('be.visible')
+        .should('have.class','-active');
+    });
+    it('click on button-site-menu2 second time should close the dropdown', () => {
+      cy.get('#button-site-menu2')
+        .click()
+        .wait(1000)
+      cy.get('#dropdown-2-menu')
+        .should('not.be.visible')
+        .should('not.have.class','-active');
+    });
+  });
+
+  describe('open and close of button-user-menu2', () => {
+    it('click on button-user-menu2 open the dropdown items', () => {
+      cy.get('#button-user-menu2')
+        .click()
+        .wait(1000)
+      cy.get('#dropdown-3-menu')
+        .should('be.visible')
+        .should('have.class','-active');
+    });
+    it('click on button-user-menu2 second time close the dropdown items', () => {
+      cy.get('#button-user-menu2')
+        .click()
+        .wait(1000)
+      cy.get('#dropdown-3-menu')
+        .should('not.be.visible')
+        .should('not.have.class','-active');
+    });
+  });
+
+  describe('Clicking on Tab link should close the drawer', () => {
+    it('Clicking on Tab link should close the drawer', () => {
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.chi-tabs li')
+        .contains('Tab link')
+        .click()
+        .wait(1000)
+      cy.get('.chi-backdrop')
+        .should('have.class','-closed')
+      cy.url().should('include', '#tab-link')
+    });
+  });
+
+  describe('check that click on "X" in menu drawer closes drawer', () => {
+    it('check  drawer closes on "X"', () => {
+      cy.get('#mobile-navigation-interaction-trigger')
+      .click()
+      .wait(1000);
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.-close')
+        .click()
+        .wait(1000)
+        .parent('.chi-drawer')
+        .should('not.have.class', '-active')
+    });
+  });
+
+  describe('Navigation to the right drawer', () => {
+    it('click on the chi-drawer__subitem-list-trigger link to second level drawer', () => {
+      cy.get('#mobile-navigation-interaction-trigger')
+        .click()
+        .wait(3000);
+      cy.get('.chi-mobile-nav__first-level-content')
+        .find('.chi-drawer__subitem-list-trigger')
+        .click()
+        .wait(1000);
+      cy.get('#second-level-1')
+        .should('have.class', '-active')
+        .should('be.visible');
+    });
+  });
+
+  describe('Click on 2nd level menus should open 3rd level drop-downs', () => {
+    it(`check Title A should open Dropdown `, () => {
+      cy.get('#second-level-1')
+        .find('.chi-mobile-nav__list li')
+        .contains('Title A')
+        .as('firstLevelElement');
```

Comment:
Can we change the declaration name? The clicked element is on the second level already (The first level would be the previous Drawer).

Reply:
done